playbooks/deprovision.yml

```diff
@@ -3,7 +3,7 @@
   gather_facts: false
   connection: local
   roles:
-  - role: ansible.kubernetes-modules
-    install_python_requirements: no
-  - role: deprovision-hello-world-apb
-    playbook_debug: false
+  - role: ansibleplaybookbundle.asb-modules
+  - role: hello-world-apb
+  vars:
```

Comment:
The spacing here and in provision.yml looks weird to me, does this work?

Reply:
It does work. Not sure what I would do to improve it.

Reply:
Ah, seeing it in the email cleared it up. Thought the var was attached to the role include and not the play.